tests/test_app.py

```diff
@@ -64,13 +64,15 @@ def aio_wrapper(*args, **kwargs):
     return real_decorator
 
 
-def mock_auth_app():
-    application = app.app_factory(config)
+async def mock_auth_app(config: configparser.ConfigParser, mock_username: str):
```

Comment:
There are a number of problems with this test setup, including mocking things that shouldn't necessarily be mocked, and this particular function not actually using variables that were passed to its invoking wrapper.

Anyway, it's probably worth a refactor of some sort but not in this PR.

Reply:
See #221